lib/wikidata_sparql.js

```diff
@@ -587,6 +593,7 @@ module.exports = {
         let additionalCode = "";
 
         let requiredShift = 0;
+        let adjustedShift = false;
```

Comment:
Sorry this is getting a little hard to follow, would you mind adding comments explaining the meaning of these variables and the logic here? Thanks!